lib/land/trackers/api_tracker.rb

```diff
@@ -41,9 +40,8 @@ def record_visit
       end
 
       def load
+        # create cookie prior to validating
         @cookie_id = cookie_id = request.params['cookie_id']
-        # validates the cookie is a UUID, if not sets to 'nil', from Tracker
-        validate_cookie
```

Comment:
This being called before the cookie record was created was an issue. We do not need this logic.